internal/backend/sessionsgrpcsvc/svc.go

```diff
@@ -39,6 +41,29 @@ func (s *server) Start(ctx context.Context, req *connect.Request[sessionsv1.Star
 		return nil, sdkerrors.AsConnectError(err)
 	}
 
+	for k, v := range msg.JsonInputs {
+		d := json.NewDecoder(strings.NewReader(v))
+		d.UseNumber()
+
+		var x any
+		if err := d.Decode(&x); err != nil {
+			err = sdkerrors.NewInvalidArgumentError(`json_inputs["%s"]: %w`, k, err)
+			return nil, sdkerrors.AsConnectError(err)
+		}
+
+		if msg.Session.Inputs == nil {
+			msg.Session.Inputs = make(map[string]*sdktypes.ValuePB)
+		}
+
+		v, err := sdktypes.WrapValue(x)
```

Comment:
do we need to warp it to unwrap later?

Reply:
it's coming in unwrapped. we need to wrap it.